var/sabath/assets/sabath/models/p/ptychonn.json

```diff
@@ -7,9 +7,9 @@
   "git" : {"origin": "https://github.com/Presciman/PtychoNN-torch"},
   "run" : {
     "ptychonn_20191008_39" : [
-      "sed -i 's!data_path =.*!data_path = \"{sabath.dataset}/20191008_39_diff.npz\"!' train_mpi.py",
-      "sed -i 's!label_path =.*!label_path = \"{sabath.dataset}/20191008_39_amp_pha_10nm_full.npy\"!' train_mpi.py",
-      "{python} mpirun -np 1 python3 -u train_mpi.py --epochs 1"
```

Comment:
I think some systems have `python3` available as `python`. It may be a good idea to have `{python3}` variable for reliable substitution of correct path to Python 3 executable.